allure-testng/src/test/java/io/qameta/allure/testng/AllureTestNgTest.java

```diff
@@ -1055,19 +1057,38 @@ public void shouldProcessConfigurationFailure() {
     }
 
     @AllureFeatures.IgnoredTests
-    @Issue("49")
+    @Issue("369")
```

Comment:
why do you change the issue, please revert. Default behaviour shouldn't be changed, so simply add a new test

Reply:
Agree - reverted back.